packages/strapi-admin/services/role.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  /**
+   * Create and save a role in database
+   * @param attributes A partial role object
+   * @returns {Promise<role>}
+   */
+  create(attributes) {
+    return strapi.query('role', 'admin').create(attributes);
+  },
+
+  /**
+   * fetch a role in database
+   * @param params query params to find the role
+   * @returns {Promise<user>}
+   */
+  fetch(params) {
```

Comment:
Maybe consider renaming to `findOne` & `find` to stay consistent with the query layer and the other services? (or perhaps this is a new convention?)